src/io/flutter/editor/FlutterPubspecNotificationProvider.java

```diff
@@ -29,6 +29,12 @@
 public class FlutterPubspecNotificationProvider extends EditorNotifications.Provider<EditorNotificationPanel> implements DumbAware {
   private static final Key<EditorNotificationPanel> KEY = Key.create("flutter.pubspec");
 
+  @NotNull private final Project project;
+
+  public FlutterPubspecNotificationProvider(@NotNull Project project) {
+    this.project = project;
```

Comment:
Ah, I see.  In that case I'm confused how the constructor gets called.  But I see that `SdkConfigurationNotificationProvider` works that way so, assuming you've verified that this works, LGTM!